tests/settings.py

```diff
@@ -97,3 +97,5 @@
 # https://docs.djangoproject.com/en/1.6/howto/static-files/
 
 STATIC_URL = '/static/'
+
+DJANGO_EJABBERD_BRIDGE_LOG='/usr/local/var/log/django-bridge.log'
```

Comment:
this is environment specific. probably put the log in the project directory by default?

Reply:
yes but its just for test @orhan89 , surely we still need this for this local project ?